src/error_chain.rs

```diff
@@ -54,6 +58,10 @@ macro_rules! error_chain_processed {
         }
 
     ) => {
+        use ::std::fmt::Debug;
+        use ::std::error::Error as StdError;
+        create_super_trait!(Trait: Debug, StdError, Send $(, $bound)*);
```

Comment:
This is a limitation of the `create_super_trait` macro currently. I will work on getting it fixed.